File: fakeLoop.py
#!/usr/bin/env python

import sys
from faker import Faker
fake = Faker()


def main():
    if len(sys.argv) <= 1:
        print "How many?"
    else:
        for i in range(0, int(sys.argv[1])):
            print (str(i) + " " + fake.name_male())


# main

if __name__ == '__main__':
        main()

File: fakeLoop.js
#!/usr/bin/env node

var fake = require('faker')

if (!process.argv[2]) {
  console.log("Usage: fakeLoop #");
} else {
  input = +process.argv[2];
  for (var i=0; i < input; i++) {
    console.log((i+1), fake.name.firstName());
  }
}

/*
de
de_AT
de_CH
en
en_AU
en_BORK
en_CA
en_GB
en_IND
en_US
en_au_ocker
es
fa
fr
it
ja
ko
nb_NO
nep
nl
pl
pt_BR
ru
sk
sv
vi
zh_CN
*/
